lib/plugins/inspec-compliance/lib/inspec-compliance/cli.rb

```diff
@@ -72,10 +72,9 @@ def profiles
       desc "exec PROFILE", "executes a #{COMPLIANCE_PRODUCT_NAME} profile"
       exec_options
       def exec(*tests)
-        config = InspecPlugins::Compliance::Configuration.new
-        return unless loggedin(config)
-
-        o = opts(:exec).dup
+        compliance_config = InspecPlugins::Compliance::Configuration.new
+        return unless loggedin(compliance_config)
+        o = config # This is an Inspec::Config object, provided by a helper method from Inspec::BaseCLI
```

Comment:
Could we just pass 'config' to diagnose and configure_logger to avoid assigning to a single letter variable (i know this pattern is all over this file).

Reply:
It's consistent with the other CLI subcommand implementations in https://github.com/inspec/inspec/blob/master/lib/inspec/cli.rb .